### ISSUE TITLE
Xcode 16.2 build

### ISSUE THREAD

**Issue by aehlke** [realm/realm-swift#8730]
### Problem

Please release a version built with Xcode 16.2 now that it's out

### How important is this improvement for you?

Would be a major improvement

### Feature would mainly be used with

Local Database only

**Comment by aehlke** [comment]
And please let us know what the situation with releases will be after Mongo finally stops contributing to Realm... Who is going to be authorized to cut releases?

**Comment by Touchgram** [comment]
I added a [forum question](https://www.mongodb.com/community/forums/t/will-community-branch-be-updated-with-new-realmswift-fixes-after-10-53-0/307331) as it's apparently been frozen since 10.53.0

re Xcode 16.2 a fix is needed for 16.1 or 16.2 to the crashes on iOS earlier than 15 as noted in [8708](https://github.com/realm/realm-swift/issues/8708) and [8723](https://github.com/realm/realm-swift/issues/8723)

As per my comment on 8708, using Xcode 16.2 currently still triggers the crash, but 16.0 is OK.

**Comment by aehlke** [comment]
No answer on whether we ought to fork yet

**Comment by hsoi** [comment]
@aehlke I'm another Realm user, like yourself, and I'm curious what exactly is the problem that you're reporting. I ask because my team is also concerned about the future of Realm and we too are hoping the company will soon provide more clear guidance and information about Realm's future.

But in the meantime… exactly what problem are you reporting with Realm and Xcode 16.2? I ask because I just built Realm (still using 10.53.1 right now) and it built fine, and a quick runtime check of our app and it seems OK. But with the state of Realm being what it is, yes we're cautious with every upgrade… so I'm curious if you're reporting/seeing some new problem due to Xcode 16.2 that perhaps we should all be aware of? Thanks.

**Comment by aehlke** [comment]
@hsoi no problem from my end at all except that I like to use verified builds from their CI for binary frameworks (it's much faster to build during dev) and usually they have an update out very quickly for new xcode releases. but of course no longer, so I'm unaware of what's next - I guess we get no community CI access, so someone will have to set it up to do it more automatically to do the releases which seem to be manual now for realm. I guess we all need to figure out how to coordinate or someone just needs to start building and sharing that with the community...

**Comment by hsoi** [comment]
@aehlke Ah gotcha. I see what you're after.

I really do hope the company comes forward soon with a concrete and articulated plan for what exactly the future will be for Realm. As it is now the silence is only serving to generate questions, uncertainty, and ill-will.

**Comment by hsoi** [comment]
Hey @aehlke I just saw this… https://github.com/realm/realm-swift/releases/tag/v10.54.2

**Comment by Touchgram** [comment]
@hsoi note that 10.54.2 isn't marked as fixing [8708](https://github.com/realm/realm-swift/issues/8708) or [8723](https://github.com/realm/realm-swift/issues/8723) so the use of Xcode 16.2 with < iOS15 is presumably still a problem.

**Comment by hsoi** [comment]
@Touchgram yup - diffing the release shows it's truly just a Xcode 16.2 binaries release (no true code changes).

Still, point is that SOMETHING is still going on… @tgoyne did work here (and I totally understand if @tgoyne can't publicly comment). On the one hand, it's nice to see something happening. On the other hand, it only serves to raise more questions about what exactly is going on and what the future holds.

**Comment by tgoyne** [comment]
I am no longer actively working on Realm outside of paid support cases, but updating the version numbers in the CI config file and pressing the release button is a fairly trivial activity.

**Comment by aehlke** [comment]
@tgoyne thx for clarifying - no community members will receive access to this repo though right? so the community ought to now rally around a new specific fork

**Comment by tgoyne** [comment]
If there are people who are interested in maintaining Realm but do not want to sign Mongo's CLA then forking would be necessary. Otherwise it's just a question of if enough people make PRs that I'm not longer able to keep up with them in my free time.
